pkg/providers/amifamily/ami.go

```diff
@@ -119,7 +119,9 @@ func (p *DefaultProvider) List(ctx context.Context, nodeClass *v1beta1.EC2NodeCl
 	}
 	amis.Sort()
 	if p.cm.HasChanged(fmt.Sprintf("amis/%s", nodeClass.Name), amis) {
-		logging.FromContext(ctx).With("ids", amis, "count", len(amis)).Debugf("discovered amis")
+		logging.FromContext(ctx).With(
+			"ids", pretty.Slice(lo.Uniq(lo.Map(nodeClass.Status.AMIs, func(a v1beta1.AMI, _ int) string { return a.ID })), 25),
```

Comment:
Does uniq maintain ordering?

Reply:
Yes, uniq will maintain the order. Only the first occurrence of a string is kept in the array. https://github.com/samber/lo/tree/master?tab=readme-ov-file#uniq